examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>category</th>\n",
+       "      <th>code_display</th>\n",
+       "      <th>status</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>[\"survey\"]</td>\n",
+       "      <td>[\"CURRENT ANTIRETROVIRAL DRUGS USED FOR TREATM...</td>\n",
+       "      <td>final</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>[\"survey\"]</td>\n",
+       "      <td>[\"LOINC Code\"]</td>\n",
+       "      <td>final</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>[\"survey\"]</td>\n",
+       "      <td>[\"TESTS ORDERED\"]</td>\n",
+       "      <td>final</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>[\"survey\"]</td>\n",
+       "      <td>[\"TESTS ORDERED\"]</td>\n",
+       "      <td>final</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>[\"survey\"]</td>\n",
+       "      <td>[\"TUBERCULOSIS PROPHYLAXIS PLAN\"]</td>\n",
+       "      <td>final</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     category                                       code_display status\n",
+       "0  [\"survey\"]  [\"CURRENT ANTIRETROVIRAL DRUGS USED FOR TREATM...  final\n",
+       "1  [\"survey\"]                                     [\"LOINC Code\"]  final\n",
+       "2  [\"survey\"]                                  [\"TESTS ORDERED\"]  final\n",
+       "3  [\"survey\"]                                  [\"TESTS ORDERED\"]  final\n",
+       "4  [\"survey\"]                  [\"TUBERCULOSIS PROPHYLAXIS PLAN\"]  final"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "runner.to_dataframe(\n",
+    "    obs.select(\n",
```

Comment:
Please include `id` in this too.

Reply:
Unresolving this too; I am guessing that you forgot to push one commit maybe? There are a few more examples too.